### PR TITLE
:green_heart: create nav component - Issue-2

### LINKED ISSUE #2: Level 2 Component : Navigation Component (Desktop)
##  Project is complete when:

- [ ] Navigation looks like figma mockup -- https://www.figma.com/file/SYCaUO0bX527hsjRt1ZioC/EDD-Help-Page?node-id=24%3A194

- [ ] All links apart from Help Center disabled

- [ ] Help Center is automatically active

### PR DESCRIPTION
**Pull request by utk0730**
Fixes #2 

- [x] Up to date with 'dev' branch
- [ ] Branch name follows issueNumber-issueName syntax
- [ ] All PR Status checks are successful
- [ ] Peer reviewed and approved

Question? See the [Contributions guidelines](https://github.com/hannahlivnat/Ask-EDD-Project/blob/main/contributing.md).

### CODE REVIEW

**Comment by utk0730** [comment]
@hannahlivnat Let me know if i have followed the project pattern.

File: constants/nav_links.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+export const navRoutes = {
```
Comment (hannahlivnat) [inline constants/nav_links.js:1]: I like the way you set up this file, super maintainable!

File: styles/navstyle.module.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    & > img {
+      height: 100%;
+    }
+  }
+  ul {
+    height: 100%;
+    list-style: none;
+    display: flex;
+    align-items: center;
+    @media (max-width: 768px) {
```
Comment (hannahlivnat) [inline styles/navstyle.module.scss:19]: good thinking ahead in adding the @media, sets us up for switching over to toggle when moving into mobile styling

**Review by hannahlivnat — APPROVED** [review]
This looks really nice! Merging

File: styles/navstyle.module.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      display: none;
+    }
+    li {
+      height: 100%;
+      padding: 0rem 1rem;
+      margin-right: 2rem;
+      display: flex;
+      align-items: center;
+      a {
+        pointer-events: none; // not a good way to disable link, but since link doesn't have a disable attr like button and to aviod onclick handler on each link, using this method
```
Comment (hannahlivnat) [inline styles/navstyle.module.scss:29]: This definitely works, there's also a great css-tricks article on handling this -- https://css-tricks.com/how-to-disable-links/

File: styles/navstyle.module.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
+.navWrapper {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0px 9px 22px rgba(71, 71, 71, 0.1); // discuss moving color , boxshadow values in global css
```
Comment (hannahlivnat) [inline styles/navstyle.module.scss:6]: good idea, there could be  main variables (box shadow, colors, fonts) used consistently across components contained in global css

File: styles/navstyle.module.scss (hunk 3 above)
Comment (utk0730) [reply styles/navstyle.module.scss:29]: noted this one. 
Thanks.

**Comment by utk0730** [comment]
@hannahlivnat I was waiting for your feedback, thanks for a detailed one.
Picking up other 2 assigned issues today.

**Pull request merged**